.circleci/config.yml

```diff
@@ -250,6 +252,19 @@ jobs:
       - store_artifacts:
           path: ~/testresults/k8s_integration_tests.html
 
+  int_test_lint:
```

Comment:
@keitwb Did you want to add this job to the workflow?

Reply:
Yes